<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(244, 220, 199);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbTitulo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>681</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 22pt &quot;Engravers MT&quot;;

color: rgb(179, 173, 86);</string>
    </property>
    <property name="text">
     <string>TIPOS DE TRIÁNGULOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbLado1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Script MT Bold&quot;;</string>
    </property>
    <property name="text">
     <string>LADO 1:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtLado1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbLado2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Script MT Bold&quot;;</string>
    </property>
    <property name="text">
     <string>LADO 2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbLado3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Script MT Bold&quot;;</string>
    </property>
    <property name="text">
     <string>LADO 3:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Lucida Calligraphy&quot;;
background-color: rgb(255, 144, 89);
</string>
    </property>
    <property name="text">
     <string>LIMPIAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAyuda">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>290</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Lucida Calligraphy&quot;;
background-color: rgb(255, 144, 89);
</string>
    </property>
    <property name="text">
     <string>AYUDA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTipo">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Lucida Calligraphy&quot;;
background-color: rgb(170, 170, 127);</string>
    </property>
    <property name="text">
     <string>DETERMINAR TIPO</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbImg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtLado2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtLado3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>190</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbResultado">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Yu Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>&gt; TIPO DE TRIÁNGULO: Desconocido</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
